console/db/migrations/20241205103951_2.1.0.sql

```diff
@@ -20,6 +20,39 @@ WHERE extension_name IN (
     'timescaledb'
 );
 
+-- Adds shared_cpu BOOLEAN field to cloud_instances for Hetzner which has both Shared and Dedicated vCPU
```

Comment:
```suggestion
-- Adds shared_cpu BOOLEAN field to cloud_instances
```
This applies not only to Hetzner